conductor-client/src/main/kotlin/com/openlattice/users/Auth0SyncTask.kt

```diff
@@ -1,77 +1,29 @@
-/*
- * Copyright (C) 2018. OpenLattice, Inc.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- *
- * You can contact the owner of the copyright at support@openlattice.com
- *
- *
- */
```

Comment:
Why are you removing this?

Reply:
Its not actually being removed, I just split the Auth0SyncTask into a Local and a Default implementation and made Auth0SyncTask into an interface. So this moved to the Default implementation. I can update my IDE settings to add this to new classes